ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -56,7 +56,7 @@ The following data model keys have been removed from `eos_designs` in v5.0.0.
 | old key 2(defs_adapter_config) | new key(TODO) |
 | old key 3.1(defs_node_type) | new key(TODO) |
 | old key 3.2(defs_node_type) | new key(TODO) |
-| old key 4(defs_svi_settings) | new key(TODO) |
+| svi_settings.ipv6_address_virtual | svi_settings.ipv6_address_virtuals |
```

Comment:
mention the data-model names here

Reply:
same as this - https://github.com/aristanetworks/avd/pull/4319#discussion_r1703981795

Reply:
```suggestion
| <network_services_key>[].vrfs[].svis[].ipv6_address_virtual | <network_services_key>[].vrfs[].svis[].ipv6_address_virtuals |
| svi_profiles[].ipv6_address_virtual | svi_profiles[].ipv6_address_virtuals |

```